test/cookie-consent.test.js

```diff
@@ -9,16 +9,8 @@ describe("CookieConsent", () => {
 
   const cookieConsent = document.createElement("cookie-consent");
 
-  cookieConsent.cookies = COOKIES;
-
   test("updatePreference", () => {
-    const input = [
-      { id: "foo", accepted: false },
-      { id: "bar", accepted: true },
-    ];
-
-    cookieConsent.updatePreference(input);
-
-    expect(cookieConsent.preferences.getAll("foo")).toEqual(input);
+    cookieConsent.updatePreference(COOKIES);
+    expect(cookieConsent.preferences.getAll()).toEqual(COOKIES);
```

Comment:
The test included code that didn't actually do anything. When the module is initialized, it actually doesn't set any localStorage values about cookies, that only happens when there has been interaction with the module. 

So the test is also valid by just calling `updatePreferences` a single time.